frontend/src/components/stylists/activeAppointmentView.jsx

```diff
@@ -42,28 +42,30 @@ function ActiveAppointmentView(props) {
   };
 
   const fetchActiveReservationsForUser = async (stylist) => {
-    try {
-      let response = await axios.get(
-        props.backendDomain +
-          "stylist/" +
-          stylist.id +
-          "/reservation?status=IP",
-        {
-          headers: {
-            Authorization: "JWT " + localStorage.getItem("token"),
-          },
-        }
-      );
+    if(stylist.role == 1){
```

Comment:
What is the reason for this change?

Reply:
ActiveAppointment was not working with Manager. It was trying to get the current appointments for the manager getting a los of 404s. Didnt see a way to switch it to others stylists like the reservation view does so it seems this only refers to stylists not managers so made the change to hide it if a manager logs in